.config-schema.json

```diff
@@ -83,6 +83,26 @@
                     "description": "The maximum number of cache keys that the storage cache can store before evicting old keys.",
                     "type": "integer",
                     "default": 100000
+                },
+                "maxOpenConns": {
+                    "description": "The maximum number of open connections to the datastore.",
+                    "type": "integer",
+                    "default": "database/sql default"
```

Comment:
People using OpenFGA might not be coming from a Go background and have knowledge of `database/sql`. Let's just be explicit with what the default is based on the `database/sql` implementation.

Reply:
How writing something like:
```
"default": "2 - the database/sql default"
```
If we say `2` and the underlying implementation changes that may not be very good. For example, the docs for database/sql say:
```
// The default max idle connections is currently 2. This may change in
// a future release.
```